Cargo.toml

```diff
@@ -36,3 +36,5 @@ crates-index = "0.18.9"
 human-panic = "1.0.3"
 bugreport = "0.5.0"
 itertools = "0.10.5"
+cargo_toml = "0.13.0"
+toml = "0.5.9"
```

Comment:
Oh, I didn't see the `toml_edit` dependency. But from what I see, it is used only in `src/manifest.rs` and only to retrieve data from a manifest. So I guess it would be possible to use `cargo_toml` to do this task, similar to what I did in the serialization.

Reply:
Worth looking into for sure!

Reply:
In https://github.com/obi1kenobi/cargo-semver-check/pull/173/commits/49ba171cc069614405a6b202d073c19e11c14f97 I've replaced the parsing in `src/manifest.rs` with `cargo_toml`. It made the code much shorter :)